ai_flow/model/internal/contexts.py

```diff
@@ -0,0 +1,50 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from ai_flow.model.context import Context
+from ai_flow.model.task_execution import TaskExecution
+from ai_flow.model.workflow import Workflow
+from ai_flow.model.workflow_execution import WorkflowExecution
+
+
+class WorkflowContext(Context):
```

Comment:
What's the point of having `WorkflowContext`, instead of making `WorkflowContextImpl` extends `Context` directly?